libyul/YulStack.cpp

```diff
@@ -108,15 +102,10 @@ void YulStack::translate(YulStack::Language _targetLanguage)
 		return;
```

Comment:
Is ``translate`` even useful anymore? I kind of doubt it given that the function does nothing now :-).

Reply:
😅